recipes/soci/all/test_v1_package/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+cmake_minimum_required(VERSION 3.1)
+project(test_package CXX)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup(TARGETS)
+
+find_package(SOCI REQUIRED CONFIG)
```

Comment:
We can trim down this file to us the improved version https://github.com/conan-io/conan-center-index/blob/master/docs/package_templates/cmake_package/all/test_v1_package/CMakeLists.txt

and delete the duplicated source code in this folder as well 🙏

Reply:
Ah thanks for the pointer, done 👍